Neos.Flow/Classes/Package/BootableInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+namespace Neos\Flow\Package;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Core\Bootstrap;
+
+/**
+ * An interface to indicate a bootable package
+ */
+interface BootableInterface
```

Comment:
&lt;nitpick&gt;I'd go for `BootablePackageInterface`&lt;/nitpick&gt;

Reply:
Yeah, can change if you want.